### PR TITLE
RED-83102 support redb multinamespace labeling

### PR DESCRIPTION
**Pull request by michalrabinowitch**

### CODE REVIEW

File: multi-namespace-redb/README.md [hunk 1]
```diff
@@ -58,10 +58,10 @@ roleRef:
 
 ```
 
-### 3. Updating the operator configmap
-
-The operator has to be deployed with a comma separated list of namespaces it will watch for REDB objects.
+### 3. Updating the managed namespaces
```
Comment (zcahana) [inline multi-namespace-redb/README.md:61]: Might be a good idea to mention that these two methods are mutually exclusive.
I'm not sure we would have a nice way to raise a visible error in this case (REC status?), but maybe we explicitly define precedence between the two methods if configured together.

Comment (michalrabinowitch) [reply multi-namespace-redb/README.md:61]: done

**Comment by yuvallevy2** [comment]
@michalrabinowitch is this still relevant?

**Comment by michalrabinowitch** [comment]
@yuvallevy2 - no this should be covered by the upcoming release promote job - closing.

**Pull request closed without merging**